### ISSUE TITLE
Context menu with GTK3

### ISSUE THREAD

**Issue by lumiru** [gnome-integration-team/firefox-gnome#404]
Hello,

Since GTK3 in Firefox, the context menu with this theme became transparent on my Ubuntu Gnome 15.10.

![capture d ecran de 2016-04-29 01-12-19](https://cloud.githubusercontent.com/assets/4085705/14904317/c34852b4-0da7-11e6-8d98-72e3a9d58be9.png)

**Comment by lumiru** [comment]
This bug only happens with custom GTK+ theme (Elementary in that case).

**Comment by elmat0** [comment]
I don't suppose you found a solution to this? Other than changing GTK+ themes obviously.

**Comment by lumiru** [comment]
Indeed. I download Elementary theme from [its daily build](https://code.launchpad.net/~elementary-os/+recipe/egtk-daily) and, for this theme and the bug is fixed.
Previously, I added a custom style sheet with [Stylish](https://addons.mozilla.org/en-US/firefox/addon/stylish/) to set a simple background color.

**Comment by elmat0** [comment]
Thank you! That did the trick.
